src/plugins/vis_default_editor/public/components/controls/utils/use_handlers.ts

```diff
@@ -36,10 +37,18 @@ function useSubAggParamsHandlers(
 
   const onAggTypeChange = useCallback(
     (aggId, aggType) => {
-      const parsedAgg = subAgg.toJSON();
+      const parsedAgg = subAgg.serialize();
```

Comment:
thank you for removing deprecated  subAgg.toJSON(); but please do the same for the one more place in that file 🙏

Reply:
done